docs/configure-and-manage/configure/qlight-node.md

```diff
@@ -0,0 +1,54 @@
+# Configure a GoQuorum qlight node
```

Comment:
Same comment as the previous one. I suggest the simple following title as the page has sections for node and client:
```suggestion
# Configure GoQuorum qlight
```
And it's a whole system in the end as you don't use client without node or node without client. So simply talking about "GoQuorum qlight" implies both node and client and their interaction. Just a suggestion though.